src/main/java/Controller/GameController.java

```diff
@@ -69,14 +76,104 @@ public int randomDice()
         int max = 6;
         return random.nextInt(max - min + 1) + min;
     }
-    public void playerRelativeWalk()
+    public boolean playerRelativeWalk()
     {
-        Player player = data.currentPlayers.get(data.currentPlayerIndex);
-        PositionImpactor.relativeMove(data, player, randomDice());
+
+        boolean flag = PositionImpactor.relativeMove(data, randomDice());
+        System.out.println("Player " + data.currentPlayer.getUserId() + " moved to " + data.currentPlayer.getPosition());
```

Comment:
the user may be confused about what the block is if just show the id. For example, when I move to block 106, I am confused about the command "Enter a block id:"

Reply:
we will add a markdown file to show users the corresponding  block to each id